elements/openlayers-core/ol-map.ts

```diff
@@ -19,6 +19,24 @@ import { forwardEvents } from './lib/events.js'
  * </ol-map>
  * ```
  *
+ * ### Setting the map container height
+ *
+ * It is expected that the map element will be wrapped in another element on your page.
```

Comment:
Not sure if this is quite helpful. Technically any element has a parent, even if that's `<body>`...